docs/decisions/20230516-hierarchical-deterministic-key-generation-algorithm.md

```diff
@@ -2,7 +2,7 @@
 
 - Status: accepted
 - Deciders: Jesus Diaz Vico, Ezequiel Postan, Pat Losoponkul, Yurii Shynbuiev
-- Date: 2023-05-16
+- Date: 2024-06-06
```

Comment:
Not sure what is the implication on rendered web and the filename. I'd add the revision date and link to the PR below.

Reply:
Good point, updated